tests/subgraph/minter-suite/minter-suite-mapping.test.ts

```diff
@@ -86,19 +86,33 @@ import {
   RegisteredNFTAddress as HolderV1RegisteredNFTAddress,
   UnregisteredNFTAddress as HolderV1UnregisteredNFTAddress
 } from "../../../generated/MinterHolderV1/MinterHolderV1";
+import {
+  AllowedHoldersOfProjects as HolderV2AllowedHoldersOfProjects,
+  RemovedHoldersOfProjects as HolderV2RemovedHoldersOfProjects,
+  RegisteredNFTAddress as HolderV2RegisteredNFTAddress,
+  UnregisteredNFTAddress as HolderV2UnregisteredNFTAddress
+} from "../../../generated/MinterHolderV2/MinterHolderV2";
+import { DelegationRegistryUpdated as MinterHolderV2DelegationRegistryUpdated } from "../../../generated/MinterHolderV2/IFilteredMinterHolderV1";
+import { DelegationRegistryUpdated as MinterMerkleV3DelegationRegistryUpdated } from "../../../generated/MinterMerkleV3/IFilteredMinterMerkleV1";
```

Comment:
nit: I think we can remove the V2 and V3 from these import-as names, since we are importing from the interfaces.